src/Model/lbsObjectCommunicator.py

```diff
@@ -115,21 +83,43 @@ def summarize_unidirectional(self, direction, print_indent=None):
             volumes.append(v)
 
             # Report current communication item if requested
+            # TODO: is this indentation really needed?
```

Comment:
I do not remember having created it. Please decide what looks best in the output and chose the better option. Tx

Reply:
sorted